CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules")
 if (UNIX)
    if (APPLE)
       if (LLVM_DIR STREQUAL "" OR NOT LLVM_DIR)
-         set(LLVM_DIR "/usr/local/Cellar/llvm@4/4.0.1/lib/cmake/llvm")
+         set(LLVM_DIR "$ENV{HOME}/opt/llvm/lib/cmake/llvm")
```

Comment:
LLVM_DIR is a cached variable does this change work correctly on a preexisting checkout?

Reply:
Previous checkouts didn't have the ~/opt symlink

Reply:
Does it work though? You've tried it? LLVM_DIR is a cached variable which typically means you can't change it through a simple `set()`

Reply:
Great question: My goal was to have mac as close as possible to how Linux; We can avoid having if/else in the CMake files looking for llvm by using a symlink from ~/opt/llvm. I also believe that using llvm instead of wasm for the folder name is easier for developers to understand. Especially now that we don't install it using experimental arguments on cmake.